### PR TITLE
Dockerize gogen-avro

### PR DESCRIPTION
**Pull request by rikonor**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+ENV binary gogen-avro
+
+WORKDIR /app
+
+RUN update-ca-certificates --fresh
+
+ADD $binary /app/
+
+ENTRYPOINT ["./$binary"]
```
Comment (bonds0097) [inline Dockerfile]: I should really update `go-skel` (or you can!) but this won't work. Turns out interpolation does not work for `ENTRYPOINT` or `CMD` in dockerfiles.

**Review by bonds0097 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (rikonor) [reply Dockerfile]: Ah gotcha, fixed here (and in `go-skel` too)

**Review by bonds0097 — APPROVED** [review]
Assuming you've run the docker container, looks good to me.

**Comment by rikonor** [comment]
Yup, tested and it seems to work fine. Squasherging.

**Pull request merged**